server/service/microsoft_mdm.go

```diff
@@ -936,18 +998,7 @@ func (svc *Service) GetMDMMicrosoftDiscoveryResponse(ctx context.Context, upnEma
 		return nil, ctxerr.Wrap(ctx, err, "resolve enroll endpoint")
 	}
 
-	// Only adding STS Auth endpoint if the UPN email is provided
```

Comment:
@gillespi314 forgot to mention, I removed the STS auth response here, which means that the STS auth endpoint will not be used for now.

Reply:
Great! Should we remove from `handler` as well?
```
	// This endpoint is unauthenticated and is used by Microsoft devices to retrieve the opaque STS auth token
	neWindowsMDM.GET(microsoft_mdm.MDE2AuthPath, mdmMicrosoftAuthEndpoint, SoapRequestContainer{})
```